File: server/Models/Keep.cs
using System.ComponentModel.DataAnnotations;

namespace keepr.Models;

public class Keep
{
    public int Id { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
    public string Name { get; set; }
    public string Description { get; set; }
    [Url] public string Img { get; set; }
    public int Views { get; set; }
    public string CreatorId { get; set; }
    public Creator Creator { get; set; }
    public VaultKeep Kept { get; set; }

    public int VaultKeepId { get; set; }

}

File: client/src/models/Keep.js
export class Keep {
    constructor(data) {
        this.id = data.id
        this.createdAt = data.createdAt
        this.updatedAt = data.updatedAt
        this.name = data.name
        this.description = data.description
        this.img = data.img
        this.views = data.views
        this.creatorId = data.creatorId
        this.creator = data.creator
        this.kept = data.kept
    }
}